roles/confluent.kafka_connect/templates/connect-distributed.properties.j2

```diff
@@ -1,4 +1,8 @@
 # Maintained by Ansible
+{% if "rest.extension.classes" in kafka_connect.properties and rbac_enabled|bool %}
+{% set rest_extension_classes = kafka_connect.properties['rest.extension.classes'] %}
+{% set kafka_connect_properties = kafka_connect.properties.pop('rest.extension.classes') %}
```

Comment:
@domenicbove maybe we could set line 2 as return value of the `pop` in line 3

Reply:
oh i see what youve done so the pop removes it from the properties! thats nifty